types/vscode-notebook-renderer/index.d.ts

```diff
@@ -0,0 +1,49 @@
+// Type definitions for non-npm package vscode-notebook-renderer 1.47
+// Project: https://github.com/microsoft/vscode-docs/blob/notebook/api/extension-guides/notebook.md
+// Definitions by: Connor Peet <https://github.com/me>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/vscode-notebook-renderer/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me

Reply:
Huh. That came from the generator, not sure how that happened 🤔